hugo/layouts/partials/sprotty-head.html

```diff
@@ -1,6 +1,23 @@
 <title>{{ .Title }}</title>
 <meta name="viewport" content="width=device-width">
 <meta charset="utf-8">
+<meta name="description" content="Diagram Vizualization Tools for Your Web Applications.
+Eclipse Sprotty is an open-source project enabling the creation of powerful diagramming tools and graphical editors.">
+<meta property="og:title" content="Eclipse Sprotty">
+<meta property="og:description" content="Diagram Vizualization Tools for Your Web Applications.
```

Comment:
WDYT of using the first line of what's in the description as title? The name "Sprotty" is already visible in the image.